tests/RobotFramework/tests/customizing/tedge_init.robot

```diff
@@ -0,0 +1,84 @@
+*** Settings ***
+Resource    ../../resources/common.resource
+Resource    ../config_management/child_conf_mgmt_plugin.robot
+Library    ThinEdgeIO
+
+Test Tags    theme:cli    theme:configuration
+Suite Setup            Custom Setup
+Suite Teardown         Custom Teardown
+
+*** Test Cases ***
+
+Check existence of init directories
+    [Documentation]    During Custom Setup these folders were removed
+    ...    this test step is confirming the deletion of the folders
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge/mosquitto-conf
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge/operations
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge/plugins
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge/device-certs
+    ThinEdgeIO.Directory Should Not Exist    /var/tedge
+    ThinEdgeIO.Directory Should Not Exist    /var/log/tedge
+
+Tedge init and check creation of folders
+    [Documentation]    Running tedge init should create these folders
+    ...    this test step is confirming the creation of the folders
+    Execute Command    sudo tedge init
+    ThinEdgeIO.Directory Should Exist    /etc/tedge
+    ThinEdgeIO.Directory Should Exist    /etc/tedge/mosquitto-conf
+    ThinEdgeIO.Directory Should Exist    /etc/tedge/operations
+    ThinEdgeIO.Directory Should Exist    /etc/tedge/plugins
+    ThinEdgeIO.Directory Should Exist    /etc/tedge/device-certs
+    ThinEdgeIO.Directory Should Exist    /var/tedge
+    ThinEdgeIO.Directory Should Exist    /var/log/tedge
+
+Check ownership of the folders
+    [Documentation]    Running tedge init has created the folders assigning default user/group
+    ...    this test step is confirming the default values for user/group
+    Check Owner of Directory    /etc/tedge    tedge:root
```

Comment:
Weird that the owner is `tedge:root` and not `tedge:tedge`. Will look into that.